app/models/spree/product_decorator.rb

```diff
@@ -0,0 +1,20 @@
+module Spree
+  module ProductDecorator
+    def self.prepended(base)
+      def base.ascend_by_postcode
+        #     # sort vendors in the order closest to the given postcode
+        vend_array = Spree::Vendor.near([52.3863596, 16.8195344])
```

Comment:
I think we have agreed to pass the location as an argument to the `ascended_by_postcode` method. I don't think having a hardcoded or default value is appropriate.